resources/views/account/partials/queries/customer-address.graphql

```diff
@@ -0,0 +1,14 @@
+{
+  customer {
+    addresses {
+      firstname
+      lastname
+      street
+      city
+      postcode
+      country_code
+      default_billing
+      default_shipping
+    }
+  }
+}
```

Comment:
Made this one since we didn't need everything in the overview.graphql

Reply:
Does it perform better? Otherwise the default is fine with me